Sources/Build/Triple.swift

```diff
@@ -113,7 +114,8 @@ public struct Triple {
     }
 
     public static let macOS = try! Triple("x86_64-apple-macosx10.10")
-    public static let x86Linux = try! Triple("x86_64-unknown-linux")
+    public static let x86_64Linux = try! Triple("x86_64-unknown-linux")
+    public static let i686Linux = try! Triple("i686-unknown-linux")
```

Comment:
i386 in the target triple would be far more conventional; most distributions including Debian/Ubuntu are using this

Reply:
Question being, does this triple get used to lookup binaries? If so, which ones? One of the reasons for using i686 because Debian Stretch reports i686, not i386, as the arch which causes all sorts of fun in the swift build. And many tools target i686 in the install.

Reply:
If this is internal, then yes, i386 should probably be used here instead.